.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,25 @@
+name: Check PR Title
```

Comment:
We've found that a very convenient way to use conventional commits and therefore generate meaningful changelogs with proper versioning is to use "squash commits" when merging with the commit being the title.